infrastructure/terraform/ansible/playbooks/update-ubuntu-migrate.yml

```diff
@@ -0,0 +1,65 @@
+# builtin modules https://docs.ansible.com/ansible/latest/collections/ansible/builtin/index.html
+---
+- hosts: qtrees_server
+  vars_files:
+    - "{{ inventory_dir }}/variables.yml"
+  become: true
+  tasks:
+  - name: Run update
+    apt:
+      update_cache: true
+  - name: Upgrade all packages
+    apt: 
+      name: "*"
+      state: latest
+  - name: Update qtrees-ai-data repo
+    become_user: ubuntu
+    shell: |
+      cd /home/ubuntu/qtrees-ai-data
+      git checkout {{ lookup('ansible.builtin.env', 'GIT_BRANCH') }}
+      git pull origin {{ lookup('ansible.builtin.env', 'GIT_BRANCH') }}
+#  - name: Update dependencies of qtrees
+#    become_user: ubuntu
+#    shell: |
+#      miniconda3/bin/conda update conda -y
+#      miniconda3/bin/conda env update --file qtrees-ai-data/requirements.yaml --prune
+#    ignore_errors: true
+  - name: stop postgrest docker
+    become_user: ubuntu
+    shell: |
+      . /home/ubuntu/setup_environment.sh
+      docker-compose -f qtrees-ai-data/infrastructure/database/docker-compose.yml down
+  - name: migrate-update
+    become_user: ubuntu
+    shell: |
+      . /home/ubuntu/setup_environment.sh
+      cd qtrees-ai-data/infrastructure/database
+      migrate -database postgresql://postgres:${POSTGRES_PASSWD}@${DB_QTREES}:5432/qtrees\?sslmode=disable -path migrations up 
```

Comment:
wofür `?sslmode=disable`

Reply:
Without it the connection didn't work, and I could figure out how to make it work without the ``` ?sslmode=disable```. Is it a big problem?